bsp/qemuarma15/qemuarma15-preempt-rt.scc

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: MIT
```

Comment:
We don't yet support rt, so fine to drop this file.

Reply:
I wonder if dropping will affect future merges from yocto?